lib/evaluate.js

```diff
@@ -82,6 +86,9 @@ export const evaluate = async ({
 
 export const runFraudDetection = async (roundIndex, measurements, { fetchRoundDetails, recordTelemetry }) => {
   const roundDetails = await fetchRoundDetails(roundIndex, recordTelemetry)
+  // IMPORTANT: The format string MUST NOT have any placeholder for the roundDetails object
```

Comment:
```suggestion
  // Omit the roundDetails object from the format string to get nicer formatting
```

Less screamy plus explains why